frame/democracy/src/lib.rs

```diff
@@ -659,6 +677,25 @@ pub mod pallet {
 			Ok(())
 		}
 
+        #[pallet::weight(T::WeightInfo::promote())]
+        pub fn promote(
+            origin: OriginFor<T>,
+            proposal_hash: T::Hash,
+        ) -> DispatchResult {
```

Comment:
When adding a new extrinsic to a pallet, it is important that you place it at the bottom of the list of extrinsics, else you will modify the extrinsic ordering, and break the transaction format.

Reply:
This has been resolved